docs/src/development/local/lando.md

```diff
@@ -4,43 +4,190 @@ sidebarTitle: "Lando"
+? Which project? <project name>
 ```
 
-That will use your existing git clone and add the appropriate Lando configuration.
+{{< /codetabs >}}
 
-Either way, start the local environment by running:
+Once the command ran successfully, you get the following:
 
-```bash
+``` bash
+   _  __                       _
+  / |/ /__ _    __  _    _____( )_______
+ /    / _ \ |/|/ / | |/|/ / -_)// __/ -_)
+/_/|_/\___/__,__/  |__,__/\__/ /_/  \__/
+
+  _________  ____  __ _______  _______  _      ______________ __  ___________  ______
+ / ___/ __ \/ __ \/ //_/  _/ |/ / ___/ | | /| / /  _/_  __/ // / / __/  _/ _ \/ __/ /
+/ /__/ /_/ / /_/ / ,< _/ //    / (_ /  | |/ |/ // /  / / / _  / / _/_/ // , _/ _//_/
+\___/\____/\____/_/|_/___/_/|_/\___/   |__/|__/___/ /_/ /_//_/ /_/ /___/_/|_/___(_)
+
+Your app has been initialized!
+
+Go to the directory where your app was initialized and run lando start to get rolling.
+Check the LOCATION printed below if you are unsure where to go.
+
+Oh... and here are some vitals:
+
+ NAME      <project name>
+ LOCATION  /Users/<your user>/<path to your project>
+ RECIPE    platformsh
+ DOCS      https://docs.lando.dev/config/platformsh.html
+```
+
+Once it completes, run `lando start` to start your app and services.
+
+``` bash
 $ lando start
+Let's get this party started! Starting app <app name>...
+...
 ```
 
-You can then download data from the environment for your current branch and load it into your local Lando environment by running:
+Once the command ran successfully, you get the following:
+
+``` bash
+
+   ___                      __        __        __     __        ______
+  / _ )___  ___  __ _  ___ / /  ___ _/ /_____ _/ /__ _/ /_____ _/ / / /
+ / _  / _ \/ _ \/  ' \(_-</ _ \/ _ `/  '_/ _ `/ / _ `/  '_/ _ `/_/_/_/
+/____/\___/\___/_/_/_/___/_//_/\_,_/_/\_\\_,_/_/\_,_/_/\_\\_,_(_|_|_)
 
-```bash
-lando pull -r database -m web/sites/default/files
+
+Your app has started up correctly.
+Here are some vitals:
+
+ NAME          <project name>
+ LOCATION      /Users/<your user>/<path to your project>
+ SERVICES      <list of your service's names>
+ APP URLS      http://localhost:62177
+               http://<app name>.lndo.site/
+               https://<app name>.lndo.site/
+               http://www.<app name>.lndo.site/
+               https://www.<app name>.lndo.site/
 ```
 
-Where the value after `-r` is the relationship name in the application for the service you want to download, and `-m` is the Platform.sh mount path you want to download.  You may specify as many relationships (multiple `-r` switches) and as many mounts (multiple `-m` switches) as you wish.
+Access your app and services by opening the `APP URLS` in your browser.
+
+For more examples, see the [getting started page](https://docs.lando.dev/platformsh/getting-started.html).
+
+## Configuring lando
+
+Lando configuration is located in [the `.lando.yml` file](https://docs.lando.dev/platformsh/config.html).
```

Comment:
What would I maybe want to configure? If there's not something specific here, then I'm not sure this section needs to be here. It could just be in a general What's next? section